common/scala/build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
     api "io.kamon:kamon-datadog_${gradle.scala.depVersion}:2.1.12"
 
     // for etcd
-    api "com.ibm.etcd:etcd-java:0.0.21"
+    api "com.ibm.etcd:etcd-java:0.0.13"
```

Comment:
Seems we can't use higher version forever?

Reply:
As I said above, we can upgrade both etcd client and akka-grpc module.